<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedbackWidget</class>
 <widget class="QWidget" name="FeedbackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>250</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="5">
    <widget class="QListView" name="lvFeedbacks">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="tbFeedbacksRemove">
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="tbFeedbackAdd">
     <property name="icon">
      <iconset theme="list-add">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>157</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="tbFeedbacksClear">
     <property name="icon">
      <iconset theme="edit-clear-list">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="tbFeedbackEdit">
     <property name="icon">
      <iconset theme="document-edit"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
